app/src/main/cpp/DeviceDelegate.h

```diff
@@ -11,8 +11,11 @@
 #include "Device.h"
 #include "ControllerDelegate.h"
 #include "GestureDelegate.h"
+#include "VRLayer.h"
 
 #include <memory>
+#include <functional>
+#include <jni.h>
```

Comment:
What needs `jni.h` in this file?

Reply:
It was there for some initial API test, but I moved everything to VRLayer.h. It's not needed anymore.